task/sast-go/0.1/sast-go.yaml

```diff
@@ -53,8 +53,15 @@ spec:
                namespace: "default",
                successes: 0,
                note: (if $tmp_not_skipped=="0" then $SKIP_MESSAGE else "" end),
-               failures: (.runs[].results // [])|map(.message.text) | unique
+               failures: (.runs[].results | length)
              }' gosec_output.json || true)
+
+          # Log out the failing runs
+          if [ $(echo $HACBS_TEST_OUTPUT | jq '.failures') -gt 0 ]
+          then
+            echo "The sast-go test fails with the following runs:"
+            echo $(jq '.runs[].results' gosec_output.json)
```

Comment:
Maybe we can use the `.runs[].results // [])|map(.message.text)` jq filter here to display only the error names.